apps/condo/domains/meter/schema/Meter.test.js

```diff
@@ -548,6 +550,73 @@ describe('Meter', () => {
             expect(meters).toHaveLength(0)
         })
 
+        test('employee with division limited role: can read meters only from his divisions', async () => {
```

Comment:
Please, introduce a `describe` blocks for employee with and without division scoped visibility.